doc/nrf/known_issues.rst

```diff
@@ -1919,18 +1923,18 @@ Multiprotocol Service Layer (MPSL)
 DRGN-16642: If radio notifications on ACTIVE are used, MPSL might assert
```

Comment:
Two already fixed issues of MPSL could be added here:

- "rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0

DRGN-17851: When a Bluetooth role was running, controller might assert due to an MPSL issue
  When  a Bluetooth role was running, the controller would assert because of an issue in MPSL."

- "rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0

DRGN-18105: Controller might abandon a link due to an MPSL issue
  The controller would abandon a link because of an issue in MPSL, causing a disconnect on the remote side."